### PR TITLE
Issue/48 - Fixed banner message not showing when AffiliateWP is not installed

### PR DESCRIPTION
**Pull request by darvinsilveira-am**
Issue #48 

Steps to test:

1) Install Affiliate Shortcodes add-on;

You should see a banner message like the image below telling you that the add-on requires AffiliateWP:

image

(Observation: the message changes if the AffiliateWP is listed as a plugin or not)

2) Now install AffiliateWP, the message should go away.

image

**Housekeeping**

* [x] Issue number has been linked below the description
* [x] Branch name follows the `issue/###` or `release/###` format
* [x] PR title is prefixed with the branch name (issue/### - Title)

**Proposed changes**

* [x] Code has been tested with no errors
* [x] Code meets stated minimum requirements
* [x] Code should (usually) not contain unrelated changes
* [x] Code style (including docs) and organization is generally consistent with the rest of the codebase
* [x] Before and after screenshots have been posted to document visual changes
* [x] Clear steps for testing the PR have been posted (as needed)

### CODE REVIEW

**Comment by amdrew** [comment]
Looking good so far

## ✅ Notice shows when AffiliateWP is not activated

image

## ✅ Notice disappears once AffiliateWP is activated

image

## ✅ Using an AffiliateWP version less than 2.6  shows inline minimum requirements notice

image

**Review by amdrew — CHANGES_REQUESTED** [review]
We'll need to update this [doc URL](https://github.com/awesomemotive/affiliatewp-affiliate-area-shortcodes/blob/master/affiliatewp-affiliate-area-shortcodes.php#L163)  to `https://affiliatewp.com/docs/minimum-requirements-roadmap/`

**Comment by amdrew** [comment]
As per the [requirements](https://affiliatewp.com/docs/minimum-requirements-roadmap/) for "All Other Official Free & Pro Add-ons" we'll need to also change the minimum version of PHP to 7.4 (It says 7.0 on the table but I believe we decided on 7.4) for this addon here: https://github.com/awesomemotive/affiliatewp-affiliate-area-shortcodes/blob/issue/48/includes/lib/affwp/class-affiliatewp-requirements-check-v1-1.php#L60

File: affiliatewp-affiliate-area-shortcodes.php [hunk 1]
```diff
@@ -105,11 +105,8 @@ protected function load() {
 			// Bootstrap to plugins_loaded.
 			$affwp_version = get_option( 'affwp_version' );
```
Comment (amdrew) [inline affiliatewp-affiliate-area-shortcodes.php:106]: `$affwp_version` is no longer being used here.

**Comment by amdrew** [comment]
@darvinsilveira Just some small changes listed above 👍

File: affiliatewp-affiliate-area-shortcodes.php (hunk 1 above)
Comment (darvinsilveira-am) [reply affiliatewp-affiliate-area-shortcodes.php:106]: Well noted

**Comment by darvinsilveira-am** [comment]
@amdrew all fixed 👍

**Review by amdrew — APPROVED** [review]

**Comment by amdrew** [comment]
I'm going to merge this in for further testing. Thanks @darvinsilveira!

**Pull request merged**